### DISCUSSION TITLE
react autocomplete 패키지

### DISCUSSION THREAD

**Discussion by hyoniiii** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#36] [General]
https://www.npmjs.com/package/react-autocomplete-hint

<img width="750" alt="image" src="https://user-images.githubusercontent.com/97004730/200729488-9a6c5554-677d-4da4-a6b6-9dc8b8312ebf.png">

검색하다 보니 이런 것도 나오던데 써도 상관없는 걸까요..

**Comment by yeondooo** [comment]
오! 데모사이트 보니까 `const options = ["orange", "banana", "apple"]` 이런식으로 디폴트 값을 넣어두고 쓰는건가봐요.
